### ISSUE TITLE
[Bug]: Using langfsue will increase the return time.

### ISSUE THREAD

**Issue by langgg0511** [BerriAI/litellm#1511]
### What happened?

After using langfuse under acompletion, the response needs to be returned after langfuse processing.

```
def test_langfuse_logging_async(): 
    try: 
        async def _test_langfuse(langfuse):
            if langfuse:
                litellm.success_callback = ["langfuse"]
            litellm.set_verbose = False
            return await litellm.acompletion(
                custom_llm_provider='openai',
                model="gpt-3.5-turbo-1106",
                messages=[{"role": "user", "content":"say 1"}],
                stream=False
            )
        print("use langfuse response time")
        start_time = time.time()
        response = asyncio.run(_test_langfuse(True))
        end_time = time.time()
        print(f"openai _response_ms: {response['_response_ms']}")
        print(f"litellm _response_ms: {(end_time-start_time) * 1000}")
        print(f"response: {response}")

    except litellm.Timeout as e: 
        print(e)
    except Exception as e: 
        print(e)
test_langfuse_logging_async()
```

### Relevant log output

```shell
use langfuse response time
openai _response_ms: 1615.442
litellm _response_ms: 2810.2331161499023
response: ModelResponse(id='chatcmpl-8iZjqcluKsOkTehVwXca9wWDZlywL', choices=[Choices(finish_reason='stop', index=0, message=Message(content='1', role='assistant'))], created=1705634502, model='gpt-3.5-turbo-1106', object='chat.completion', system_fingerprint='fp_c596c86df9', usage=Usage(completion_tokens=1, prompt_tokens=10, total_tokens=11), _response_ms=1615.442)

not use langfuse response time
openai _response_ms: 1280.494
litellm _response_ms: 1282.829999923706
response: ModelResponse(id='chatcmpl-8iZkQDgG4tMRKn2S5Dyey41Re1hsV', choices=[Choices(finish_reason='stop', index=0, message=Message(content='1', role='assistant'))], created=1705634538, model='gpt-3.5-turbo-1106', object='chat.completion', system_fingerprint='fp_c596c86df9', usage=Usage(completion_tokens=1, prompt_tokens=10, total_tokens=11), _response_ms=1280.494)
```

**Comment by langgg0511** [comment]
litellm==1.18.3

**Comment by ghost** [comment]
We've reached out to the langfuse team about this, but i would also expect logging to happen independent of the completion call. I'll see what's happening here - thanks for raising this @langgg0511

**Comment by ishaan-jaff** [comment]
I can repro this error too, goes away when I remove this line: 

https://github.com/BerriAI/litellm/blob/c5e144af23ec1ae345b9a70cfcce122a4a82c67e/litellm/utils.py#L2360

**Comment by ishaan-jaff** [comment]
Using this fixes the issue for me - will open a PR 
```python
  threading.Thread(
      target=logging_obj.async_success_handler, args=(result, start_time, end_time)
  ).start()
```

**Comment by ishaan-jaff** [comment]
PR here: https://github.com/BerriAI/litellm/pull/1519

**Comment by langgg0511** [comment]
> PR here: #1519

@ishaan-jaff thanks

**Comment by langgg0511** [comment]
The issue has been resolved in version v1.18.8.
